packages/cli/src/zosfiles/compare/ds/Dataset.handler.ts

```diff
@@ -0,0 +1,116 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession, IHandlerParameters, ITaskWithStatus, TaskStage, TextUtils } from "@zowe/imperative";
+import { Get, IZosFilesResponse } from "@zowe/zos-files-for-zowe-sdk";
+import { ZosFilesBaseHandler } from "../../ZosFilesBase.handler";
+import { diff } from "jest-diff";
+
+/**
+ * Handler to view a data set's content
```

Comment:
Should this say something about comparing two datasets?

Reply:
I spotted that yesterday. I have it fixed on my local drive, I will push the change.